src/main/resources/schemas/schema-form.json

```diff
@@ -2,9 +2,16 @@
   "type" : "object",
   "id" : "urn:jsonschema:io:gravitee:policy:JsonValidationPolicyConfiguration",
   "properties" : {
+    "scope" : {
+      "title": "Scope",
+      "description": "Execute policy on <strong>request</strong> or <strong>response</strong> phase.",
+      "type" : "string",
+      "default": "REQUEST",
+      "enum" : [ "REQUEST", "RESPONSE" ]
+    },
     "errorMessage" : {
       "title": "Http error message",
-      "description": "Http error message to send when request is not valid. Status code is 400 as Bad request.",
+      "description": "Http error message to send when request is not valid. Status code is 400 as Bad request for REQUEST scope. Status code is 500 as Internal Error for RESPONSE scope (without straight respond mod).",
```

Comment:
Why 500 for RESPONSE ?
Not sure there is any internal server error here...

Reply:
Validation problems on the response phase can appear because of the outdating schema, if somebody caught the response and changed that, any backend problems which affect response messages and so on. Anyway, this case is definitely not about client errors. That`s why 500. Maybe you have better idea for that?

Reply:
No better idea my side, it's definitely not a client error.
The only thing that bothers me is that we can think that 500 (internal server error) is due to the gateway, and I'm not sure that's the case (and it's not!)
But there is no other status code which can reflect better this kind of scenario...

Reply:
Can you please explain problems which can arise there, I do not fully understand your worries.

Reply:
Sorry, did not read myself...

So, most of the time, I'm assuming that 500 status code is coming from the last server during request processing, in our case the API gateway.
What I'm just trying to explain is that, I don't want that users think that the issue is coming from the API gateway because they receive a 500 internal server error status code.
But as I said in my previous comment, there is no other status code which can be used instead...

Reply:
Okay, I understand you, but yes I do not see another alternative here too.